cypress/e2e/create_tx.cy.js

```diff
@@ -77,6 +79,41 @@ describe('Queue a transaction on 1/N', () => {
     })
     cy.contains('Estimated fee').should('exist')
 
+    // Asserting the relayer option is present
+    cy.contains('Via relayer').find('[type="radio"]').should('be.checked')
+    cy.contains('Sponsored by Gnosis Chain')
+    cy.get('span').contains('Estimated fee').next().should('have.css', 'text-decoration-line', 'line-through')
+    cy.contains('Via relayer').contains(/[0-5] of 5/)
+
+    cy.contains('Estimated fee').click()
+    cy.contains('Edit').click()
+    cy.contains('Owner transaction (Execution)').parents('form').as('Paramsform')
+
+    // Only gaslimit should be editable when the relayer is selected
+    const arrayNames = ['userNonce', 'maxPriorityFeePerGas', 'maxFeePerGas']
+    arrayNames.forEach((element) => {
+      cy.get('@Paramsform').find(`[name="${element}"]`).should('be.disabled')
+    })
+    cy.get('@Paramsform')
+      .find('[name="gasLimit"]')
+      .clear()
+      .type('300000')
+      .invoke('prop', 'value')
+      .should('equal', '300000')
+    cy.get('@Paramsform').find('[name="gasLimit"]').parent('div').find('[data-testid="RotateLeftIcon"]').click()
```

Comment:
Same as comment above. Would be preferred to select the input based of visible text: either the label or the placeholder.

Reply:
same as before